packages/core/src/renderers/renderer.util.tsx

```diff
@@ -153,6 +153,14 @@ export const mapStateToControlProps = (state, ownProps) => {
     required
   };
 };
+
+export const mapDispatchToControlProps = dispatch => ({
+  handleChange(path, value) {
```

Comment:
do we have a case where the path of the handleChange is a different one than the one in the properties?
I think we only have the case where we pass in the path from the properties to the handleChange method.
We could then get rid of the path parameter by providing two parameter to the mapDispatchToControlProps as the second one could be the props: `mapDispatchToProps(dispatch, [ownProps]): dispatchProps`

Reply:
Not sure if I got this right, but the relevant `path` for updating the respective data bit of a control is determined is `mapStateToProps`, isn't it?